scalameta/dialects/src/main/scala/scala/meta/dialects/package.scala

```diff
@@ -173,6 +181,7 @@ package object dialects {
     def allowOrTypes = Scala211.allowOrTypes
     def allowTraitParameters = Scala211.allowTraitParameters
     def allowLiteralTypes: Boolean = false // Scheduled to be included in 2.12.2
+    def allowTrailingComma: Boolean = true // Schedule to be included in 2.12.2
```

Comment:
Good question. I think both should be true, but don't want to change allowLiteralTypes in this PR.